test/meson.build

```diff
@@ -14,6 +14,20 @@ td += executable('readdir_inode', 'readdir_inode.c',
                  include_directories: include_dirs,
                  install: false)
 
+td += custom_target('test746.so',
```

Comment:
Is there a reason we can't use Meson's standard rules for building shared objects (instead of custom_target)?

Reply:
For shared libraries to work properly overloading symbols when preloaded you need specific parameters. I am not sure if Meson's standard building rules won't interfere with the (I hardly use Meson).
I'll give it a try and see if it works.

Reply:
Got it working with Meson's shared_library(). ChatGPT to the rescue. Patch coming.